spyder/plugins/explorer/tests/test_explorer.py

```diff
@@ -8,12 +8,19 @@
 Tests for explorer.py
 """
 
-# Test library imports
+# Standard imports
+import os
+import os.path as osp
+
+# Third party imports
 import pytest
 
 # Local imports
 from spyder.plugins.explorer.widgets import (FileExplorerTest,
-                                             ProjectExplorerTest)
+                                             ProjectExplorerTest, QMessageBox)
```

Comment:
Please import `QMessageBox` like this

`from qtpy.QtWidgets import QMessageBox`